src/components/list-item-actions/list-item-actions.tsx

```diff
@@ -0,0 +1,29 @@
+import { List } from '@patternfly/react-core';
+import React from 'react';
+import { StatefulDropdown } from '../patternfly-wrappers/stateful-dropdown';
+
+interface IProps {
+  kebabItems?: any[];
+  buttons?: any[];
```

Comment:
```suggestion
  kebabItems?: React.ReactNode[];
  buttons?: React.ReactNode[];
```

(I think :) .. although you may have to resort to `(React.ReactNode | undefined)[]` if it complains the array items can be undefined/null)